Design-Process.md

```diff
@@ -0,0 +1,194 @@
+by an LDT member.
+
+### Steps to move a [Discussion](#proposed-feature) to a [Champion feature](#championed-feature)
+
+When an LDT member decides to champion a discussion, they take the following steps:
+
+1. Create a new proposal champion issue.
+    * If preferred, the LDT member can ask the original proposer to create this issue.
+    * Note: it can be easier to create the PR for step 6 first, get that into a ready-to-merge state, and then create the champion issue at that point, depending on the complexity
+      of the feature.
+2. Assign themselves to the champion issue.
+3. Apply the `Proposal Champion` label to the new issue, as well as to the original discussion.
+4. Link to the original discussion from the champion issue.
+5. Lock the proposal champion issue for comments to ensure that discussion continues in the discussion area, rather than on the champion issue.
+6. Fill out and check in a [proposal template](proposals/proposal-template.md) for the feature. Exact specese is not required, but there should be enough detail to have a
+   meaningful triage session.
+    * This is also something the LDT member can ask a community member to open a PR for, if they are willing.
+    * The filled out proposal should include a link to the champion issue for easy navigation.
+
+### Bringing open questions to LDM
+
+During the course of development of a feature, there are several different types of questions that need to get brought to LDM for answers. The most important overriding factor
+for any question is that there is a checked-in commit that contains the question. The document and commit will be linked as part of the notes so that future readers of the notes
+can understand the full context in which the question was asked.
+
+#### Alternative proposals, supplemental documentation
+
+As part of the initial design of a feature, a number of different proposals may be brought as part of the design process, either as alternatives to an initial design, or as
+supplemental materials to an existing design to help drive conversation in LDM. We want to keep these "supplemental" materials in one place, rather than scattered throughout
+the repo as different issues, discussions, and other documents. For such material, they should go in the [working group folder](meetings/working-groups/) for that feature. Not
+all features will have such a folder; indeed, most will not. For these documents, please check them in _before_ bringing them to an LDM. The LDM organizer should be able to
+link to an exact document, not to a PR.
+
+#### Specific implementation questions
+
+During the implementation process, we will often come up with specific scenarios that need to be brought to an LDM and discussed. These questions should be placed in the proposal
+specification, in an `Open Questions` section below the main specification text. Each question should have a _linkable_ header, such that the notes that go over the question can
```

Comment:
Curious: do you intend for LDM notes to point to the linkable header at the tip of the document or do a permalink? Basically when I go back ~3 years, open an LDM note and click through the open questions will I see the document at the current state or the state ~3 years ago?

Reply:
You should see the document as it was 3 years ago, so you can understand the context in which the question was asked. It could very well be that the question no longer even makes sense, given the current state of the proposal.